CONTRIBUTING.md

```diff
@@ -118,7 +117,7 @@ is for providing user information to client side game when developing.
         * `minecraft_access.accesswidener`: Yet another configuration.
         * `minecraft_access-common.mixins.json`: Mixin configuration that tells the Mixin framework where to find Mixin classes under `common` directory.
         * `assets/minecraft_access`: Contains custom resource files of this mod, structured in the [Minecraft Resource Pack](https://minecraft.wiki/w/Resource_pack) format.
-            * `lang`: The I18N text used by this mod for narration purpose, managed with [Weblate].
+            * `lang`: The I18N text used by this mod for narration purpose, managed with [Weblate](https://hosted.weblate.org/engage/minecraft-access/).
```

Comment:
This is not necessary, the URL is already at the end of the file

Reply:
But, it's quoted with [ ]...

Reply:
In markdown you can use `[text]` and then put `[text]: url` at the end of the file, this is equivalent to `[text](url)`. I generally prefer this syntax because it makes lines significantly shorter and thus greatly inproves readability.

Reply:
TIL!